README.md

```diff
@@ -14,56 +14,27 @@ Checks of dependencies are executed in parallel, to ensure one slow dependency d
 
 ```json
 {
-    "results": {
-        "http": [
-            {
-                "name": "Error",
-                "success": false,
-                "duration": 0.294109529,
-                "error": {
-                    "err": "some error"
-                }
-            },
-            {
-                "name": "Not 2..",
-                "success": false,
-                "duration": 0.562796634,
-                "httpStatus": "404 Not Found"
-            },
-            {
-                "name": "Google",
-                "success": true,
-                "duration": 0.667266497
-            }
-        ],
-        "redis": [
-            {
-                "name": "Error",
-                "success": false,
-                "duration": 0.016504383,
-                "error": {
-                    "err": "some error"
-                }
-            },
-            {
-                "name": "Redis",
-                "success": true,
-                "duration": 0.311321392
-            }
-        ],
-        "sql": [
-            {
-                "name": "Postgres Database",
-                "success": true,
-                "duration": 1.1117e-05
-            },
-            {
-                "name": "Microsoft SQL Database",
-                "success": true,
-                "duration": 0.342005575
-            }
-        ]
-    }
+ "GitHub": {
```

Comment:
Don't mind most of the changes to the response. However having the dependency name duplicated as the key to each object, as well as the value of the name field may be hard to consume? 

This is what I'm meaning: 

```suggestion
 {
  "name": "GitHub",
  "type": "http",
  "success": true,
  "duration": 0.4550456,
  "Reason": null
 },
```

Reply:
Name + Type ?

Reply:
`json:"-"` ?